sphinx/source/Concepts.rst

```diff
@@ -17,10 +17,17 @@ The Basics
 -----------
 
 If you've found this project, you probably already know a few things about ISO 11783 (ISOBUS) or J1939.
-It's OK if you haven't heard of those before, but this concepts overview assumes you know a bit about CAN already.
+It's OK if you haven't heard of those before, but this concepts overview assumes you know a bit about `CAN bus <https://en.wikipedia.org/wiki/CAN_bus>`_ already.
 
 Lets discuss some of the basics you'll need to know to use this library.
 
+.. image:: ../images/concepts_1.png
+   :width: 126px
+   :align: left
+   :alt: Minimal Block Diagram
+
+The above block diagram shows the basic components of an ISOBUS module, a **Hardware API** that handles the CAN transceiver driver and raw CAN frames, a **Networking API** that handles Control Functions for listening to data and sending to other modules, also transport handling of messages. The **Application** is where you will write your code to handle the data you receive and send.
+
```

Comment:
I really like diagrams like yours, they really help explain something words alone can't! 

But my instinct tells me that the concepts page might not really be the right place for it. Instead I would suggest placing the diagram, together with the explanation, at the top of the first tutorial. I feel like that will overwhelm the user less, as really the concepts is more about the theory than the implementation.